props/prop_base.h

```diff
@@ -17,6 +17,11 @@
 #undef SAVE_VOLUME
 #endif
 
+#ifndef AUDIO_CLASH_SUPPRESSION_LEVEL
+// Account for Audio Volume in Clash Detection (range 1 ~ 50)
+#define AUDIO_CLASH_SUPPRESSION_LEVEL 1
```

Comment:
If the default value is this low, is there really any point?

Reply:
I was originally thinking you'd want it to be almost nothing if the define was missing to keep existing behavior from previous versions